book/docs/Open_formats_tools.ipynb

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
<p> </p><p>rioxarray provides support for -&gt; rioxarray simplifies</p><p>We can check but I'm pretty sure rioxarray uses rasterio, and other libraries to do the actual reading/writing </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/scaling_science/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>